eventually/static/src/containers/progress/Progress.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import LinearProgress from 'material-ui/LinearProgress';
+
+export const lightGreen500 = '#8bc34a';
+
+const style={
```

Comment:
please add whitespaces at both sides near to equal char =)

Reply:
done